File: help/pt_pt/exercicios_da_ficha_3/Ficha03_011/Ficha03_011.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Número inteiro\t: " );// print text
    n1 = parseInt(prompt("Número inteiro\t: " ,"0") );
    
    document.write(n1 + "\n");// print input 
    document.write("Número real \t: " );// print text
    n2 = parseFloat(prompt("Número real \t: " ,"0.0") );
    
    document.write(n2 + "\n");// print input 
    document.write("Números Ordenados" );
    document.write("\n" + Math.min( n1 , n2 ) );
    document.write("\n" + Math.max( n1 , n2 ) );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:30

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_011/Ficha03_011.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
#define min(x, y) (((x) < (y)) ? (x) : (y))
#define max(x, y) (((x) > (y)) ? (x) : (y))
//Main Function
int main() {
    printf("Número inteiro\t: ");
    int n1;
    scanf("%d", &n1);
    printf("Número real \t: ");
    float n2;
    scanf("%f", &n2);
    printf("Números Ordenados");
    printf("\n%f",min( n1 , n2 ) );
    printf("\n%f",max( n1 , n2 ) );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:30
